alert-database/src/main/java/com/synopsys/integration/alert/database/api/StaticJobAccessor.java

```diff
@@ -210,7 +211,10 @@ public StaticJobAccessor(
                    .filter(distributionJobEntity -> !(distributionJobEntity.getBlackDuckJobDetails().getFilterByProject() &&
                                                           distributionJobEntity.getBlackDuckJobDetails().getProjectNamePattern() != null &&
                                                           !Pattern.matches(distributionJobEntity.getBlackDuckJobDetails().getProjectNamePattern(), projectName) &&
-                                                          !distributionJobEntity.getBlackDuckJobDetails().getBlackDuckJobProjects().contains(projectName))
+                                                          distributionJobEntity.getBlackDuckJobDetails().getBlackDuckJobProjects()
+                                                              .stream()
+                                                              .map(BlackDuckJobProjectEntity::getProjectName)
+                                                              .noneMatch(projectName::equals))
                    )
```

Comment:
We call `distributionJobEntity.getBlackDuckJobDetails()` times within the same filter. This is the perfect candidate for a helper method.